<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsDialog</class>
 <widget class="QDialog" name="PrefsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="intervalLabel">
           <property name="text">
            <string>&amp;Refresh interval</string>
           </property>
           <property name="buddy">
            <cstring>intervalSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="intervalSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>120</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="minutesLabel">
           <property name="text">
            <string>minutes</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="loglevelLabel">
           <property name="text">
            <string>Log le&amp;vel</string>
           </property>
           <property name="buddy">
            <cstring>loglevelComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QComboBox" name="loglevelComboBox">
           <item>
            <property name="text">
             <string>Debug</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Info</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Warning</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Error</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Critical</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="logGroupBox">
         <property name="title">
          <string>Log to &amp;file</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="logfileLabel">
              <property name="text">
               <string>File &amp;name</string>
              </property>
              <property name="buddy">
               <cstring>logfileLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="logfileLineEdit">
              <property name="text">
               <string>PySteamTrades.log</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="logfileButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="emailGroupBox">
         <property name="title">
          <string>E&amp;mail notifications</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="senderLabel">
              <property name="text">
               <string>&amp;Sender email</string>
              </property>
              <property name="buddy">
               <cstring>senderLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="senderLineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="recipientLabel">
              <property name="text">
               <string>&amp;Recipient email</string>
              </property>
              <property name="buddy">
               <cstring>recipientLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="recipientLineEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="hostLabel">
              <property name="text">
               <string>SMTP &amp;host</string>
              </property>
              <property name="buddy">
               <cstring>hostLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="hostLineEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="portLabel">
              <property name="text">
               <string>SMTP &amp;port</string>
              </property>
              <property name="buddy">
               <cstring>portLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="portLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="encryptionGroupBox">
            <property name="title">
             <string>&amp;Encryption</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="encryptionLabel">
                 <property name="text">
                  <string>Encryption t&amp;ype</string>
                 </property>
                 <property name="buddy">
                  <cstring>encryptionComboBox</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="encryptionComboBox">
                 <item>
                  <property name="text">
                   <string>SSL</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>TLS</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="loginGroupBox">
            <property name="title">
             <string>&amp;Login</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="1" column="1">
                <widget class="QLineEdit" name="passwordLineEdit">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="passwordLabel">
                 <property name="text">
                  <string>Pass&amp;word</string>
                 </property>
                 <property name="buddy">
                  <cstring>passwordLineEdit</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="usernameLabel">
                 <property name="text">
                  <string>&amp;Username</string>
                 </property>
                 <property name="buddy">
                  <cstring>usernameLineEdit</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="usernameLineEdit"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="testButton">
              <property name="text">
               <string>&amp;Test settings</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>117</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="autoSearchTab">
      <attribute name="title">
       <string>Auto search</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="autoSearchGroupBox">
         <property name="title">
          <string>&amp;Auto search</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="haveLabel">
            <property name="text">
             <string>I &amp;have:</string>
            </property>
            <property name="buddy">
             <cstring>haveTextEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="haveTextEdit">
            <property name="placeholderText">
             <string>Insert games, one per line</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="wantLabel">
            <property name="text">
             <string>I &amp;want:</string>
            </property>
            <property name="buddy">
             <cstring>wantTextEdit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="wantTextEdit">
            <property name="placeholderText">
             <string>Insert games, one per line</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>149</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>intervalSpinBox</tabstop>
  <tabstop>loglevelComboBox</tabstop>
  <tabstop>logGroupBox</tabstop>
  <tabstop>logfileLineEdit</tabstop>
  <tabstop>logfileButton</tabstop>
  <tabstop>emailGroupBox</tabstop>
  <tabstop>senderLineEdit</tabstop>
  <tabstop>recipientLineEdit</tabstop>
  <tabstop>hostLineEdit</tabstop>
  <tabstop>portLineEdit</tabstop>
  <tabstop>encryptionGroupBox</tabstop>
  <tabstop>encryptionComboBox</tabstop>
  <tabstop>loginGroupBox</tabstop>
  <tabstop>usernameLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>testButton</tabstop>
  <tabstop>autoSearchGroupBox</tabstop>
  <tabstop>haveTextEdit</tabstop>
  <tabstop>wantTextEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>PrefsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>720</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>737</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
